syntax = "proto2";

package meteo.map.sxf;

//! Путь для сохранения файлов
message SxfLoaderConf {
  required string sxf_path   = 1;
  required string rsc_path   = 2;
  repeated int32 layers = 3;
};
